### DISCUSSION TITLE
Getting error on accessing admin panel of Firefly 6.1.11

### DISCUSSION THREAD

**Discussion by jameswill** [firefly-iii/firefly-iii#8694] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

HI,

I have installed Firefly 6.1.11 on my test server with PHP 8.3 and Mysql 5.7. I am able to installed it successfully using the [documentation](https://docs.firefly-iii.org/how-to/firefly-iii/installation/self-managed/). 

I am getting "**500 Internal Server Error**" error on accessing admin panel ie. http://domain.com/[[sub-dir]]/public/register
On turning on the debug I found the following error.

**Uncaught Illuminate\Foundation\ViteManifestNotFoundException: Vite manifest not found at: [[path]]/public/build/manifest.json in [[path]]/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php:728** 

Also attaching here the log file
[ff3-fpm-fcgi-2024-03-20.log](https://github.com/firefly-iii/firefly-iii/files/14665882/ff3-fpm-fcgi-2024-03-20.log)

I feel the the installer is failing to generate /public/build/* files and hence causing the issue.

### Debug information

[2024-03-20 12:50:44] production.ERROR: Exception is: {"class":"Illuminate\\View\\ViewException","errorMessage":"Vite manifest not found at: [[path]]\/public\/build\/manifest.json (View: [[path]]\/resources\/views\/auth\/register.blade.php)","time":"Wed, 20 Mar 2024 12:50:44 +0100","file":"[[path]]\/vendor\/laravel\/framework\/src\/Illuminate\/Foundation\/Vite.php","line":728,"code":0,"version":"6.1.11","url":"http:\/\/[[domain]]\/manual\/firefly6111\/public\/register","userAgent":"Mozilla\/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko\/20100101 Firefox\/123.0","json":true,"method":"GET","post":""}  
[2024-03-20 12:50:44] production.WARNING: [Bcc] Could not email or log the error. Please validate your email settings, use the .env.example file as a guide.  
[2024-03-20 12:50:44] production.ERROR: Vite manifest not found at: [[path]]/public/build/manifest.json 

### Expected behaviour

Login page should load without any error.

### Steps to reproduce

1. php composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly-iii 6.1.11
2. Fill up DB details in .env
3. php artisan firefly-iii:upgrade-database
4. php artisan firefly-iii:correct-database
5. php artisan firefly-iii:report-integrity
6. php artisan passport:keys
7. Visit  http://domain/sub-dir/public and getting the error.

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/how-to/firefly-iii/upgrade/self-managed/#created-using-composer-create-project

Using create-project is no longer supported for the installation or upgrade of Firefly III. You'll have to follow the new instructions.

**Comment by jameswill** [comment]
I thought that is for upgrade. Do we need to follow in install as well ?

**Reply by JC5** [reply]
Yes, see https://docs.firefly-iii.org/how-to/firefly-iii/installation/self-managed/

**Comment by jameswill** [comment]
HI.

Thank you..It resolves the issue.. However few images are failing to load on admin panel.

What could be the issue for that ?
![firefly](https://github.com/firefly-iii/firefly-iii/assets/6368036/81cfc2be-d641-40eb-ab92-8f29842fa212)

**Reply by JC5** [reply]
Those icons are part of a font, and it should be included in the zip download.

If you open the developer console (ctrl-shift-i) and reload the page, do you see any 404's?

**Comment by Maxco10** [comment]
@JC5 i

The same issue on login page. 

![image](https://github.com/firefly-iii/firefly-iii/assets/6735404/54d94924-226c-4b10-a66c-fef9d0d2c814)

![image](https://github.com/firefly-iii/firefly-iii/assets/6735404/3636bd38-f91d-41bd-b939-1b4c6cca350e)

The .htaccess file is the same of repository.

Ideas?

**Reply by JC5** [reply]
I see it! I'll pick it up for the next release.

**Reply by JC5** [reply]
I see a 403 forbidden, so the file exists. Are you hosting on Azure by any chance?

**Reply by Maxco10** [reply]
@JC5 , yes i am using azure for Firefly, I have a virtual machine.

**Reply by JC5** [reply]
Alright, thanks. I see a 403 error. So the file exists, you're just not allowed to access it. I don't think I can fix that.

**Reply by Maxco10** [reply]
@JC5 
i saw it but the file and folder 's permissions are ok and I am using the htaccess of project. 
What permissions do you have for that folder?

**Reply by JC5** [reply]
Are you using Apache as a web server? 


This is from the demo:

<img width="664" alt="Screenshot 2024-03-21 at 06 28 02" src="https://github.com/firefly-iii/firefly-iii/assets/5889984/4c197457-217f-42e3-a904-9756a81f2795">

**Reply by Maxco10** [reply]
Yes, Apache. I suppose the htaccess is the same on repository, yes?

**Reply by JC5** [reply]
Indeed. The demo site unpacks the zipfile.

Could be that the extension is blocked by another part of your configuration?

**Comment by jameswill** [comment]
HI,

Getting the same issue on firefly 6.1.12 version as well.

![image](https://github.com/firefly-iii/firefly-iii/assets/6368036/8df4c7b7-645b-4489-9f53-948a7aef3ed2)

The fonts are present in /public/build/assets directory.

Also I noticed that script is trying to load the fonts from the root url and I have made an installation in a sub directory and hence it's failing to load. Is the sub directory installation discontinued ?

**Reply by Maxco10** [reply]
the answer is yes, you can open the app-def and fix the path of that resources but I believe you get my same error.

@JC5  i get my error(previous topic) after that I fixed the path, otherwise the src in css it was wrong because my firefly project is not on root folder but in subfolder.

**Reply by jameswill** [reply]
Is there any mentioning about it on the installation documentation ?

**Reply by Maxco10** [reply]
no, but it is an issue after this release.

**Reply by JC5** [reply]
I'll pick it up over the weekend, thanks for letting me know!

**Reply by Maxco10** [reply]
@JC5 @jameswill 

 I fixed the "bug" on /public/assets/app-de4f62e0.css file. The issue is on path for the font. To have a clean UI before next release we can change the file "app-de4f62e0.css" removing "build/assets/" from path.

![image](https://github.com/firefly-iii/firefly-iii/assets/6735404/83fe4d31-4783-445e-9ea6-537a6171d099)

![image](https://github.com/firefly-iii/firefly-iii/assets/6735404/777b0ad4-3616-4c5c-b041-d86f0ed51540)


_**For the next release:**_
We need to remove "build/assets/" from "src" when we generate this app-css file.

**Reply by JC5** [reply]
The next (development) release will point to `./fa-solid-900-9fc85f3a.woff2`.
